cEP-0020.md

```diff
@@ -0,0 +1,389 @@
+  - Name: Intermediate
+  - Points_required = 30
+  - Privileges:
+    - Mark your own PR as wip
+- Level-IV:
+  - Name: Cool
+  - Points_required = 50
+  - Privileges:
+    - Mark a PR by another newcomer as wip
+- Level-V
+  - Name: Awesome
+  - Points_required = 100
+  - Privileges
+    - Get assigned to difficulty medium issue
+- Level-VI
+  - Name: Cool and Awesome
+  - Points_required = 200
+  - Privileges
+    - Assign newcomers to difficulty newcomer issue
+    - Assign newcomers to difficulty low issue
+- Level-VII
+  - Name: Expert and Trusted
+  - Points_required = 350
+  - Privileges
+    - Invite newcomers to org
+    - Assign newcomers to difficulty medium issue
+
+#### Note
+
+Implementing the above privileges is beyond the scope of the gamification system.
+The gamification system will merely expose a list of privileges that each
+person has, and other systems(i.e.: [corobo](https://github.com/coala/corobo))
+are expected to adapt to use these privileges.
+
+While implementing this process, a first manual check should be done for testing
+purpose, and then automated only if there is no serious risk or there is a good
+risk reaction mechanism in place.
+
+### Badges
+
+Another way of explicitly nudging a newcomer to action is to award badges for
+completing tasks. Badges enable newcomers to follow their performance and compare
+to other newcomers.
+Badges will be awarded after performing certain activities by the newcomers:
+
+- Badge-I:
+  - Name: The bug finder
+  - Details: The one who find bugs in the existing codebase
+  - Activities:
+    - Create at least one issue which labeled as bug
+      and approved by a maintainer i.e.: had label `process/verified`
+- Badge-II:
+  - Name: The bear hunter
+  - Details: The one who create issues about new bears
+  - Activities:
+    - Create at least one bear proposal issue that also had label
+      `process/varified`
+- Badge-III:
+  - Name: The bear writer
+  - Details: The one who write new bears
+  - Activities:
+    - Merged at least 2 bear proposal PRs
+- Badge-IV:
+  - Name: The helper
+  - Details: The one who answers every newcomer questions on Gitter
+  - Activities:
+    - At least 20 good messages to newcomer
+- Badge-V:
+  - Name: The coala Developer
+  - Details: The ones who are in the coala developers team
+  - Activities:
+    - Introduce `.coafile` in other projects
+    - Merge a difficulty/newcomer Pull Request
+    - Review at least a difficulty/newcomer Pull Request
+    - Merge a difficulty/low Pull Request
+    - Review at least a difficulty/low or higher Pull Request
+
+So, for becoming a developer at coala, a newcomer must complete all the
+activities mentioned in the `coala Developer` badge. But in case a newcomer
+is not able to become a developer at coala, he/she could take away the other
+badges.
+
+#### Note
+
+Names and metrics of tracked activities will also be shown on the user
+dashboard along with the number of times they have been done. That will
```

Comment:
What's user dashboard? Maybe you need another section/paragraph to explain it a bit.